script/bundle-libraries.sh

```diff
@@ -2,13 +2,13 @@
 set -xe
 
 ./script/util/bundle-module.sh load-figure.js App.LoadFigure
-./script/util/bundle-module.sh webtest-lib.mjs Test.Util.Puppeteer --platform=node
+esbuild ./output-es/Test.Util.Puppeteer/index.js --bundle --platform=node > dist/fluid/shared/webtest-lib.js
```

Comment:
It would be good to drop a link here to the Stack Overflow article (or other online resource) where the `esbuild` solution was mentioned.